### PR TITLE
[bug 1129102] ditch ditchchart flag

### PR DESCRIPTION
**Pull request by lgp171188**
Ditch ditchchart waffle flag from the checks. Remove the flag from
the database using the provided reversible migration.

### CODE REVIEW

File: fjord/base/migrations/0003_remove_ditchchart_waffle_flag.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
```
Comment (willkg) [inline fjord/base/migrations/0003_remove_ditchchart_waffle_flag.py]: This needs a module-level docstring explaining what this migration does.

http://fjord.readthedocs.org/en/latest/db.html#creating-a-data-migration

See this as an example:

https://github.com/mozilla/fjord/blob/master/fjord/heartbeat/migrations/0004_auto_20150225_1232.py#L2

**Comment by willkg** [comment]
Other than the missing docstring, this looks good to me.

**Comment by lgp171188** [comment]
I have added the docstring to the migration file

**Comment by willkg** [comment]
Awesome! Thank you!

**Comment by willkg** [comment]
d44faf3 [bug 1129102] ditch ditchchart flag

**Pull request merged**